ios/browser/api/url/url_utils.h

```diff
@@ -52,6 +52,29 @@ OBJC_EXPORT
 - (bool)brave_hasScheme:(NSString*)scheme NS_SWIFT_NAME(hasScheme(scheme:));
 @end
 
+OBJC_EXPORT
+@interface NSURL (StaticUtilities)
+
+/// Returns the domain registry
+///  google.co.uk -> co.uk
+///  google.com -> com
++ (NSString*)brave_registryFromHost:(NSString*)host
```

Comment:
could we have some unit tests for all these methods?

Reply:
Hi. I do have unit tests for all of them in the `brave-ios` repo as part of the `NSURLExtensionTests.swift` file :)
I made sure it passes all the same tests as the original code as well as the new tests for the URL versions too.